Gemfile

```diff
@@ -77,6 +77,9 @@ group :development do
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/209319966-df407545-ebda-41f7-bb41-aa2104b37da6.png)

- [ ] Your link to your issues page is broken :fearful: Kindly fix this link to make your readme file descriptive and all the links should work :+1: